spec.md

```diff
@@ -591,13 +595,41 @@ schema being used.
 }
 ```
 
+##### Deprecation Object 
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| deprecated* | boolean | Indicates that this service or plan is deprecated and will be removed in the future. See [deprecating services and plans](#deprecating-services-and-plans) for more information. |
+| description | string | A short description of the deprecation. |
+| since | date string | ISO 8601 formatted date when the Service Broker deprecated the service or plan. |
+| eol | date string | ISO 8601 formatted date when the Service Broker expects to remove the service or plan from the catalog. |
+| alterntives | array of strings | An array of alterntives services or plans that should be concidered from the broker. This is a list of names. If the debrecated object is a service, these are service names. If the debrecated object is a plan, these are plan names within the same service. |
+
+\* Fields with an asterisk are REQUIRED.
 
 ### Adding a Service Broker to the Platform
 
 After implementing the first endpoint `GET /v2/catalog` documented
 [above](#catalog-management), the Service Broker will need to be registered
 with your Platform to make your services and plans available to end users.
 
+### Deprecating services and plans
+
+A Service Broker can signal to a Platform (through the Catalog) that a service
+or plan is deprecated. The Service Broker MAY provide optional messaging and
+alterntives for the deprecated service or plan. This gives the Platform the
+opportunity to display meaningful advice to their users to migrate away from
+the deprecated service or plan.
+
+Service Brokers MUST allow existing Service Instances to be updated as they
+were before deprecration. Service Brokers MAY block provisioning of a Service
+Instance using a deprecated service or plan, or block updating an existing
+Service Instance to a deprecated plan by return error code `DeprecatedError`
+(see [Service Broker Errors](#service-broker-errors)).
+
+The Service Broker SHOULD allow the Platform to continue to
```

Comment:
The "SHOULD allow" here and the "MAY block", while not technically in conflict because both allow a choice, as a reader they feel contradictory.  I think what you were trying to say is:
- brokers SHOULD allow platforms to continue to provision new instances using the old stuff
- platforms SHOULD encourage their users to migrate away from deprecated services/plans
- but if brokers do block it then they MUST use the DeprecatedError

Right?

Also, we should talk about Bindings too. E.g. People can still create new Bindings for deprecated services/plans.

Reply:
We decided to remove the error. If the platform wants to enforce this, then they can, it was decided that it is not the role of the broker to error on deprecated provisionings, if that is the case, just delete the service or plan.